CoreUtilities/src/au/gov/asd/tac/constellation/utilities/temporal/TemporalFormatting.java

```diff
@@ -295,10 +295,13 @@ private static void completeDateString(final StringBuilder dateTimeStringBuilder
         switch (currentLength) {
             case 0:
                 dateTimeStringBuilder.append("1970");
+                break;
```

Comment:
@formalhaut69 as tempting as it is to add `break`s here, the fall through is intentional; a way to append all subsequent date formats which I think is clever. Noted that SQ will flag it and understand its reasoning. Not sure what others think but I'd like to mark this as **wont fix**.

Note that as is this can't be imported because it breaks current functionality and the unit tests; try running `TemporalFormattingNGTest` and you will see what I mean.

Reply:
@arcturus2 Okay, no worries. Would you like me to remove these breaks and resubmit the pull request?

Reply:
Yep - you should revert this particular change and mark the SQ warnings as all good.

Reply:
@formalhaut69 @arcturus2 @cygnus-x-1 Assuming you are referring to the blocker code smell, I have already done this